[2023-01-01 03:34:19,535] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dbt_example.test.my_project.not_null_my_second_dbt_model_id manual__2023-01-01T11:33:52.445964+00:00 [queued]>
[2023-01-01 03:34:19,539] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dbt_example.test.my_project.not_null_my_second_dbt_model_id manual__2023-01-01T11:33:52.445964+00:00 [queued]>
[2023-01-01 03:34:19,539] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-01-01 03:34:19,539] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-01-01 03:34:19,539] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-01-01 03:34:19,546] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): test.my_project.not_null_my_second_dbt_model_id> on 2023-01-01 11:33:52.445964+00:00
[2023-01-01 03:34:19,547] {standard_task_runner.py:52} INFO - Started process 4546 to run task
[2023-01-01 03:34:19,551] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dbt_example', 'test.my_project.not_null_my_second_dbt_model_id', 'manual__2023-01-01T11:33:52.445964+00:00', '--job-id', '60', '--raw', '--subdir', 'DAGS_FOLDER/dbt_example.py', '--cfg-path', '/var/folders/7s/852sy8qx6lx4d2xxm8yv15sw0000gn/T/tmpm2696qug', '--error-file', '/var/folders/7s/852sy8qx6lx4d2xxm8yv15sw0000gn/T/tmporbaku4l']
[2023-01-01 03:34:19,552] {standard_task_runner.py:80} INFO - Job 60: Subtask test.my_project.not_null_my_second_dbt_model_id
[2023-01-01 03:34:19,585] {task_command.py:370} INFO - Running <TaskInstance: dbt_example.test.my_project.not_null_my_second_dbt_model_id manual__2023-01-01T11:33:52.445964+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2023-01-01 03:34:19,620] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dbt_example
AIRFLOW_CTX_TASK_ID=test.my_project.not_null_my_second_dbt_model_id
AIRFLOW_CTX_EXECUTION_DATE=2023-01-01T11:33:52.445964+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-01T11:33:52.445964+00:00
[2023-01-01 03:34:19,621] {subprocess.py:62} INFO - Tmp dir root location: 
 /var/folders/7s/852sy8qx6lx4d2xxm8yv15sw0000gn/T
[2023-01-01 03:34:19,621] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'cd /Users/kayleeyvo/Documents/DS_Learning/datafold_project/repos/demo/dbt/my_project && eval "$(pyenv init -)" && eval "$(pyenv virtualenv-init -)" && pyenv activate demo_dbt && dbt run --models not_null_my_second_dbt_model_id']
[2023-01-01 03:34:19,626] {subprocess.py:85} INFO - Output:
[2023-01-01 03:34:19,940] {subprocess.py:92} INFO - pyenv-virtualenv: prompt changing will be removed from future release. configure `export PYENV_VIRTUALENV_DISABLE_PROMPT=1' to simulate the behavior.
[2023-01-01 03:34:22,374] {subprocess.py:92} INFO - 11:34:22  Running with dbt=1.1.0
[2023-01-01 03:34:22,437] {subprocess.py:92} INFO - 11:34:22  Found 2 models, 4 tests, 0 snapshots, 0 analyses, 167 macros, 0 operations, 0 seed files, 0 sources, 0 exposures, 0 metrics
[2023-01-01 03:34:22,438] {subprocess.py:92} INFO - 11:34:22
[2023-01-01 03:34:22,438] {subprocess.py:92} INFO - 11:34:22  [[33mWARNING[0m]: Nothing to do. Try checking your model configs and model specification args
[2023-01-01 03:34:22,975] {subprocess.py:96} INFO - Command exited with return code 0
[2023-01-01 03:34:22,992] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dbt_example, task_id=test.my_project.not_null_my_second_dbt_model_id, execution_date=20230101T113352, start_date=20230101T113419, end_date=20230101T113422
[2023-01-01 03:34:23,034] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-01 03:34:23,075] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
